### DISCUSSION TITLE
OON File Out of Network Object

### DISCUSSION THREAD

**Discussion by jamessiefke** [CMSgov/price-transparency-guide#115] [General]
As part of the OON object, data is supposed to be grouped by Billing Code such as CPT code.  My understanding is that the plan would have to have 20 claims for a provider for that Billing Code in order to be in the file. 

If we use more than 1 CPT code to make the payment determination, are we only supposed to be reporting based on the primary value? 

What if there is a modifier involved?

**Comment by rakeshkatte** [comment]
@jamessiefke my understanding is - since we are only publishing the negotiated price amount and normally only 1 CPT code will be used to determine the pricing. So using multiple CPT codes to arrive at the "payment" amount might not cause any problems here.

For the modifiers, please refer to discussion #12
